environment/env.go

```diff
@@ -36,6 +36,16 @@ const (
 	CassandraPort = "CASSANDRA_DB_PORT"
 	// CassandraDefaultPort Cassandra default port
 	CassandraDefaultPort = "9042"
+	// CassandraUsername env
+	CassandraUsername = "CASSANDRA_DB_USERNAME"
+	// CassandraUsername env
+	CassandraDefaultUsername = "cassandra"
```

Comment:
let's make it back to empty. It's not just unit test, there are other integ test, pipeline and even dev envs already use empty username/password.

Changing it is going to be painful but unnecessary. I will keep it empty for simple

Reply:
nit: comments smell like copy & paste :)